File: backend/src/main/java/cohappy/backend/model/dto/request/PatchUserDTO.java
package cohappy.backend.model.dto.request;

import cohappy.backend.model.Anagrafica;
import cohappy.backend.model.Portfolio;
import lombok.Data;
import org.springframework.data.annotation.Id;

import java.time.LocalDate;
import java.util.List;

@Data
public class PatchUserDTO{
    private String userCode;

    private String name;
    private String surname;
    private LocalDate birthDate;
    private Integer age;
    private List<byte[]> images;
    private String email;
    private String phoneNumber;
    private String password;
}

File: frontend/app/src/main/java/cohappy/frontend/model/dto/request/PatchUserDTO.kt
package cohappy.frontend.model.dto.request

import java.time.LocalDate

data class PatchUserDTO(
    val userCode: String,
    var name: String? = null,
    var surname: String? = null,
    var birthDate: LocalDate? = null,
    var age: Int? = null,
    var images: List<ByteArray>? = null,
    var email: String? = null,
    var phoneNumber: String? = null,
    var password: String? = null
)